website/src/pages/index.module.css

```
@@ -233,140 +144,100 @@
 }
 
-.closing .textLink {
-  color: var(--jazz-brass-light);
+.textLink:hover {
+  color: var(--jazz-primary-accessible);
+  text-decoration: underline;
+  text-underline-offset: 0.25rem;
 }
 
-@keyframes heroEnter {
+@keyframes introEnter {
```

Stylelint (17.14.0)

[error] Expected keyframe name "introEnter" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)